### PR TITLE
ERM-2416

### PR DESCRIPTION
**Pull request by CalamityC**
* switch from eresources to packages link

### CODE REVIEW

**Review by EthanFreestone — APPROVED** [review]

**Comment by EthanFreestone** [comment]
Will merge after the rest of ERM-2416 in ui-agreements so as not to break erm-comparisons in the meantime

**Comment by CalamityC** [comment]
Solved together with this PR #181 
Do not merge this PR

**Pull request closed without merging**
